suite-native/device/src/middlewares/buttonRequestMiddleware.ts

```diff
@@ -6,7 +6,7 @@ export const prepareButtonRequestMiddleware = createMiddlewareWithExtraDeps(
     (action, { dispatch, getState, next }) => {
         next(action);
 
-        if (action.type === UI.REQUEST_PIN || action.type === UI.INVALID_PIN) {
+        if (action.type === UI.REQUEST_PIN) {
```

Comment:
because it has no buttonRequest in payload so it was deleting all buttonRequests and that is not what we want on invalid pin - that would remove also pin request...